src/matlab2tikz.m

```diff
@@ -1502,6 +1502,16 @@ case guitypes()
 
     options = setAxisTicks(m2t, options, axis, pgfTicks, pgfTickLabels, ...
         hasMinorTicks, tickDirection, isDatetimeTicks);
+        
+    options = setAxisTickLabelStyle(m2t, options,axis,handle);
+end
+% ==============================================================================
+function options = setAxisTickLabelStyle(m2t, options, axis, handle)
+    keywordTickLabelRotation = [upper(axis), 'TickLabelRotation'];
+    Rotation = getOrDefault(handle, keywordTickLabelRotation, 0);
```

Comment:
I would rather have keywordTickLabelRotation removed and the corresponding assignment directly into the getOrDefault. It is only used once and doesnt bring any additional insight.

Also it would be nice to rename Rotation to tickLabelRotation or whatever suits you best.